hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/HazelcastSqlOperatorTable.java

```diff
@@ -353,6 +358,7 @@ public SqlNode visit(SqlNodeList nodeList) {
 
         @Override
         public SqlNode visit(SqlCall call) {
+            call = expandViews(call);
```

Comment:
We run this also for CREATE VIEW statements. This means that when you execute `CREATE VIEW view2 AS SELECT * FROM view1`, then the `view1` will be inlined in the view definition for view2. We should not do this.